packages/edit-widgets/src/components/layout/index.js

```diff
@@ -12,13 +12,16 @@ import Sidebar from '../sidebar';
 import Interface from './interface';
 import UnsavedChangesWarning from './unsaved-changes-warning';
 
-function Layout( { blockEditorSettings } ) {
+function Layout( { blockEditorSettings, isInCustomizer } ) {
```

Comment:
I'm not sure how far we'll get with a boolean prop because I imagine that the design of the widgets screen and the customizer widgets editor may diverge a lot.

Maybe we should have two components which share a common base component?

Reply:
I figured maybe we should make it simple for now, and split them into different components when there're the needs. It's hard for me imagine what it would look like in the future 😆 , maybe it's just me though.